### PR TITLE
feat: (preview mode) enable pipe the console .log and .error to the parent in a iframe

### PR DESCRIPTION
**Pull request by leanmendoza**
## What does this PR change?
This enables to pipe of the console calls to an iframe parent

Editor counterpart: https://github.com/decentraland/editor-sdk7/pull/12

## Why?
https://github.com/decentraland/sdk/issues/631

### CODE REVIEW

File: browser-interface/static/preview.html [hunk 1]
```diff
@@ -537,6 +537,23 @@
...
+            const targetWindow = window.parent || window
+            targetWindow && targetWindow.postMessage({ type, payload: args }, '*')
+            originalFunctionCallback(...args)
+          }
+        }
+
+        console.log('Pipe the console log and error enabled.')
+
+        globalThis.originalConsole = { log: globalThis.console.log, error: globalThis.console.error }
+        globalThis.console.log = pipe('log', globalThis.originalConsole.log)
```
Comment (menduz) [inline browser-interface/static/preview.html:554]: I think should only pipe the scene's console from the dev adapter. Wdyt?

**Review by menduz — CHANGES_REQUESTED** [review]

File: browser-interface/static/preview.html (hunk 1 above)
Comment (leanmendoza) [reply browser-interface/static/preview.html:554]: I think we'd miss a lot of info in the editor if we only handle the scene logs.

Comment (menduz) [reply browser-interface/static/preview.html:554]: Isn't that the only thing that matters? There are toons of useless logs there, if we'd like to debug we can open vscode's own console to see the logs

Comment (leanmendoza) [reply browser-interface/static/preview.html:554]: It **should be** the only thing that matters. But this enables the possibility to filter other things that can matter. I agree the Dev-Adapter approach is the ideal one

**Review by menduz — APPROVED** [review]
amazing thank you <3

File: browser-interface/packages/lib/logger/logger.ts [hunk 2]
```diff
@@ -53,6 +53,51 @@ function createDefaultLogger(type: 'kernel' | 'unity', prefix: string, subPrefix
   }
 }
 
+export function createForwardedLogger(type: 'kernel' | 'unity', prefix: string, subPrefix: string = ''): ILogger {
+  const targetWindow = globalThis.window.parent || globalThis.window
```
Comment (eordano) [inline browser-interface/packages/lib/logger/logger.ts:57]:
```suggestion
  const targetWindow = globalThis.window?.parent || globalThis.window
```

**Review by cazala — APPROVED** [review]

**Pull request merged**